Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ebigxa/atlas-db-scxa-base:0.1.0
+FROM quay.io/ebigxa/atlas-db-scxa-base:0.15.0.0
```

Comment:
This tag doesn't have the best naming - but it does exist in quay so let's leave it as it is